tests/CI/Parser/_fixtures/invalid.yaml

```diff
@@ -0,0 +1 @@
+this: is: invalid::::
```

Comment:
This file is unused and can be safely removed. (You've inlined the contents into the `parse()` method call.)